apps/client/app/modules/photo/components/edit-title-modal/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@/components/button/index.tsx'
+import { Modal } from '@/components/modal/index.tsx'
+
+interface Props {
+	isOpened: boolean,
+	toggleModal: () => void
+}
+
+export function EditTitleModal( {isOpened, toggleModal }: Props) {
+const onClose = ()=>{
+	isOpened = false
+}
```

Comment:
```suggestion
const onClose = ()=>{
	isOpened = false
}
```

`toggleModal` could be used to close the modal so you probably don't need this